src/CLICommand/parameters_helper.ts

```diff
@@ -51,6 +53,10 @@ export class CommonContext {
 		throw new Error('No wallet file neither seed phrase provided!');
 	}
 
+	public async getWalletAddress(): Promise<string> {
+		return this.getParameterValue(DriveAddressParameter) || this.getWallet().then((wallet) => wallet.getAddress());
```

Comment:
`--drive-address` was never outlined in the man page and honestly I find its name confusing. Is it a drive entity ID? A drive tx ID? It's actually neither. It's a wallet address passed in by a user or that of a supplied wallet file. I'm going to rename this accordingly (i.e. to the `--address` option that was specified in the man page).

Reply:
I also just noticed a misspelling in the `DestinationAddressParameter` value assignment:
```
export const DestinationAddressParameter = 'destAddres'; // This is missing a trailing 's'
```
and that seems to be used in `send-ar` action. I'll fix that as well.

Reply:
af53e1c and 771aae0

Reply:
Done